tests/Doctrine/Tests/DBAL/Platforms/OraclePlatformTest.php

```diff
@@ -697,6 +697,60 @@ public function testQuotedTableNames()
     }
 
     /**
+     * @dataProvider getReturnsGetListTableColumnsSQL
+     * @group DBAL-831
+     */
+    public function testReturnsGetListTableColumnsSQL($database, $expectedSql)
+    {
+        $this->assertEquals($expectedSql, $this->_platform->getListTableColumnsSQL('"test"', $database));
+    }
+
+    public function getReturnsGetListTableColumnsSQL()
```

Comment:
Merging this regardless of the fragile test. We may simply rework the test, should some changes break this.